Plugins/AssetUsageDetector/Editor/AssetUsageDetectorSearchFunctions.cs

```diff
@@ -718,21 +718,16 @@ private ReferenceNode SearchSpriteAtlas( Object unityObject )
 					referenceNode.AddLinkTo( SearchObject( masterAtlas ), "Master Atlas" );
 			}
 
-#if UNITY_2018_2_OR_NEWER
-			Object[] packables = spriteAtlas.GetPackables();
-			if( packables != null )
-			{
-				for( int i = 0; i < packables.Length; i++ )
-					referenceNode.AddLinkTo( SearchObject( packables[i] ), "Packed Texture" );
-			}
-#else
-			SerializedProperty packables = spriteAtlasSO.FindProperty( "m_EditorData.packables" );
-			if( packables != null )
+			int length = spriteAtlas.spriteCount;
+			Sprite[] sprites = new Sprite[length];
+			spriteAtlas.GetSprites( sprites );
+
+			for (int i = 0; i < length; i++)
 			{
-				for( int i = 0, length = packables.arraySize; i < length; i++ )
-					referenceNode.AddLinkTo( SearchObject( packables.GetArrayElementAtIndex( i ).objectReferenceValue ), "Packed Texture" );
+				Texture texture = sprites[i].texture;
```

Comment:
> The clones will use the packed texture of this atlas.

If Sprite packing is **Always Enabled**, the Texture will point to the atlas texture (the combined texture) in Play mode and the references won't be found.